<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-22T09:08:04Z" name="Default suite" finished-at="2024-04-22T09:08:48Z" duration-ms="43924">
    <groups>
    </groups>
    <test started-at="2024-04-22T09:08:04Z" name="Default test" finished-at="2024-04-22T09:08:48Z" duration-ms="43924">
      <class name="sampleTestNG.TestSuit.PIMmodulePositiveTestCase">
        <test-method is-config="true" signature="LoadApplication()[pri:0, instance:sampleTestNG.TestSuit.PIMmodulePositiveTestCase@5ca1f591]" started-at="2024-04-22T14:38:04Z" name="LoadApplication" finished-at="2024-04-22T14:38:25Z" duration-ms="20815" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoadApplication -->
        <test-method signature="AddNewEmployee()[pri:0, instance:sampleTestNG.TestSuit.PIMmodulePositiveTestCase@5ca1f591]" started-at="2024-04-22T14:38:25Z" name="AddNewEmployee" finished-at="2024-04-22T14:38:35Z" duration-ms="9343" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddNewEmployee -->
        <test-method signature="EmployeeListTableDataStoring()[pri:2, instance:sampleTestNG.TestSuit.PIMmodulePositiveTestCase@5ca1f591]" started-at="2024-04-22T14:38:35Z" name="EmployeeListTableDataStoring" finished-at="2024-04-22T14:38:48Z" duration-ms="13296" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [true] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at sampleTestNG.TestSuit.PIMmodulePositiveTestCase.EmployeeListTableDataStoring(PIMmodulePositiveTestCase.java:286)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EmployeeListTableDataStoring -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:sampleTestNG.TestSuit.PIMmodulePositiveTestCase@5ca1f591]" started-at="2024-04-22T14:38:48Z" name="tearDown" finished-at="2024-04-22T14:38:48Z" duration-ms="247" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- sampleTestNG.TestSuit.PIMmodulePositiveTestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
